core/src/test/java/org/elasticsearch/indices/recovery/RecoveryTests.java

```diff
@@ -113,27 +113,10 @@ public void testRecoveryWithOutOfOrderDelete() throws Exception {
             orgReplica.applyIndexOperationOnReplica(3, 1, VersionType.EXTERNAL, IndexRequest.UNSET_AUTO_GENERATED_TIMESTAMP, false,
                 SourceToParse.source(orgReplica.shardId().getIndexName(), "type", "id2", new BytesArray("{}"), XContentType.JSON), u -> {});
 
-            final int translogOps;
+            final int translogOps = 4; // 3 ops + seqno gaps
```

Comment:
This removes the tests for the cases where we've updated the index settings. Is that ok?

Reply:
I am working to have another test for this as the assumption in this test is no longer correct.

Reply:
which other test is that?